graphene_django_plus/input_types.py

```diff
@@ -0,0 +1,70 @@
+"""
+Module that defines the mapping between model fields and GraphQL input types
+used in mutations.
+"""
+import functools
+from typing import Union
+
+from django.db import models
+from django.db.models import Field
+from django.db.models.fields.reverse_related import ForeignObjectRel
+import graphene
+from graphene import Scalar
+from graphene.types.structures import Structure
+from graphene_django.converter import convert_django_field_with_choices
+from graphene_django.registry import Registry
+
+from .types import UploadType
+
+
+@functools.singledispatch
+def get_input_field(
+    field: Union[Field, ForeignObjectRel], registry: Registry
+) -> Union[Scalar, Structure]:
+    """
+    Convert a model field into a GraphQL input type used in mutations.
+    :param field: A model field.
+    :param registry: Registry which holds a mapping between django models/fields
+      and Graphene types.
+    :return: A scalar that can be used as an input field in mutations.
+    """
+    return convert_django_field_with_choices(field, registry)
+
+
+@get_input_field.register(models.FileField)
+def get_file_field(field, registry):
+    return UploadType(
+        description=field.help_text,
+    )
```

Comment:
Codecov is pointing that this line was not covered. Since the change should be trivial, could you add a test for `FileField`/`UploadType`?

Reply:
Hi @bellini666 
I added the missing test.